Gopkg.toml

```diff
@@ -108,3 +108,11 @@
 [[constraint]]
   name = "github.com/lib/pq"
   version = "1.0.0"
+
+[[constraint]]
+  name = "github.com/Azure/azure-sdk-for-go"
+  version = "32.5.0"
+
+[[override]]
+  name = "github.com/Azure/go-autorest"
+  version = "13.0.0"
```

Comment:
the `Gopkg` files were removed and replaced with `go.mod` files. You may want to sync up your codebase with the latest version

Reply:
K will move to go.mod